commit 78a150172b301238da4366d5f58f15af86ad35cb
Author: David Tardon <dtardon@redhat.com>
Date:   Wed Oct 6 09:50:50 2010 +0200

    set REDLAND_CFLAGS for system redland.
    
    unoxml/source/rdf/makefile.mk used it for ages (since integration of CWS
    odfmetadata), but it had never been set .-)

diff --git a/configure.in b/configure.in
index 4d27ac0..282a6b5 100644
--- configure.in
+++ configure.in
@@ -5308,6 +5308,7 @@ else
 	SYSTEM_REDLAND=NO
 fi
 AC_SUBST(SYSTEM_REDLAND)
+AC_SUBST(REDLAND_CFLAGS)
 AC_SUBST(REDLAND_LIBS)
 
 dnl ===================================================================
diff --git a/set_soenv.in b/set_soenv.in
index 969a928..2e42f66 100644
--- set_soenv.in
+++ set_soenv.in
@@ -1933,6 +1933,7 @@ ToFile( "BUILD_UNOWINREG",   "@BUILD_UNOWINREG@",  "e" );
 ToFile( "USE_XINERAMA",      "@USE_XINERAMA@",     "e" );
 ToFile( "XINERAMA_LINK",     "@XINERAMA_LINK@",    "e" );
 ToFile( "SYSTEM_REDLAND",    "@SYSTEM_REDLAND@",   "e" );
+ToFile( "REDLAND_CFLAGS",    "@REDLAND_CFLAGS@",   "e" );
 ToFile( "REDLAND_LIBS",      "@REDLAND_LIBS@",     "e" );
 ToFile( "SYSTEM_HUNSPELL",   "@SYSTEM_HUNSPELL@",  "e" );
 ToFile( "HUNSPELL_CFLAGS",   "@HUNSPELL_CFLAGS@",  "e" );
